src/Compilers/Test/Core/Generated.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
BOM

Reply:
will clean these up on the next PR in this series